packages/js/src/utils/minimal-publish-script.ts

```diff
@@ -0,0 +1,44 @@
+import type { Tree } from '@nrwl/devkit';
+
+const publishScriptContent = `
+/**
+* This is a minimal script to publish your package to "npm".
+* This is meant to be used as-is or customize as you see fit.
+* 
+* This script is executed on "dist/path/to/library" as "cwd" by default.
+*
+* You might need to authenticate with NPM before running this script.
+*/
+
+import { execSync } from 'child_process';
+import { readFileSync, writeFileSync } from 'fs';
+import * as chalk from 'chalk';
+
+// Executing publish script: node path/to/publish.mjs {name} --version {version} --tag {tag}
+// Default "tag" to "next" so we won't publish the "latest" tag by accident.  
+const [, , name, version, tag = 'next'] = process.argv;
+
+// A simple SemVer validation to validate the version 
+const validVersion = /^\\d+\\.\\d+\\.\\d(-\\w+\\.\\d+)?/;
+if (!version || !validVersion.test(version)) {
+console.error(chalk.red(\`Invalid version\`));
+process.exit(1);
+}
+
+// Updating the version in "package.json" before publishing
+const json = JSON.parse(readFileSync(\`package.json\`).toString());
```

Comment:
should we test for the file before trying to read to prevent unexpected errors? I get this is a minimal script but just a thought.